### PR TITLE
add PIM for Groups to AAD baseline instructions for 7.6 through 7.9

### LINKED ISSUE #920: Modify AAD baseline instructions 7.6 through 7.9 to cover PIM for Groups
# 💡 Summary #

Per the prototyping performed for #792 we need to revise the AAD baseline instructions for policies 7.6 through 7.9 to include PIM for Groups. Currently only PIM for roles is supported. There is a coding issue #919 associated with this baseline update.

### PR DESCRIPTION
**Pull request by tkol2022**
## 🗣 Description ##

Modify the AAD baseline document instructions for policies 7.6 through 7.9 to incorporate PIM for Groups. There is a coding issue that is currently in the works that aligns with these changes as well.

Closes #920 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] Unit tests added/updated to cover PowerShell and Rego changes.
- [x] Functional tests added/updated to cover PowerShell and Rego changes.
- [x] All relevant functional tests passed.
- [ ] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [ ] PR passed smoke test check.
- [ ] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [ ] Resolved all merge conflicts on branch
- [ ] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: PowerShell/ScubaGear/baselines/aad.md [hunk 1]
```diff
@@ -649,22 +649,17 @@ Exception cases:
 
 #### MS.AAD.7.6v1 Instructions
 
-1. In **Azure Active Directory** create a new group named **Privileged Escalation Approvers**. This group will contain users that will receive role activation approval requests and approve or deny them.
+1. In **Azure AD Privileged Identity Management (PIM)**, under **Manage**, select **Azure AD roles**.
 
-2. Assign this new group to the Azure AD role **Privileged Role Administrators**. This permission allows users in this group to adjudicate role activation approval requests.
```
Comment (gdasher) [inline PowerShell/ScubaGear/baselines/aad.md:654]: Can you provide some context for the removal of the dedicated group for approvers?

**Review by gdasher — CHANGES_REQUESTED** [review]

**Review by gdasher — APPROVED** [review]
Based on offline discussion, I'm fine with these changes.

**Review by mitchelbaker-cisa — APPROVED** [review]
Went into admin console and checked the updated steps.

File: PowerShell/ScubaGear/baselines/aad.md (hunk 1 above)
Comment (tkol2022) [reply PowerShell/ScubaGear/baselines/aad.md:654]: Provided offline and the reviewer approved.

**Comment by tkol2022** [comment]
@nanda-katikaneni ready for merge

**Pull request merged**